common/tools/eslint-plugin-azure-sdk/src/rules/ts-doc-internal-private-member.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @file Rule to check if a private class member is tagged with @internal
+ * @author Hamsa Shankar
+ */
+
+import { ParserServices, TSESTree } from "@typescript-eslint/experimental-utils";
+import { Node } from "estree";
+import { ParserWeakMapESTreeToTSNode } from "@typescript-eslint/typescript-estree/dist/parser-options";
+import { Rule } from "eslint";
+import { SyntaxKind } from "typescript";
+import { getRuleMetaData } from "../utils";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+/**
+ * Helper method for reporting on a node
+ * @param node the Node being operated on
+ * @param context the ESLint runtime context
+ * @param converter a converter from TSESTree Nodes to TSNodes
+ * @param typeChecker the TypeScript TypeChecker
+ * @throws if the Node passes throught the initial checks and has an internal tag
+ */
+const reportInternal = (
+  node: Node,
+  context: Rule.RuleContext,
+  converter: ParserWeakMapESTreeToTSNode,
+): void => {
+  const tsNode = converter.get(node as TSESTree.Node);
```

Comment:
nitpick: you can cast to `any` here then don't need to do that later in two places.
```suggestion
  const tsNode = converter.get(node as TSESTree.Node) as any;
```

Reply:
I suggested this other way instead 😆 . Ultimately it's no big deal to me but I think it's slightly preferable not to store `any`s in variables and instead cast. If `tsNode` is `any` here, then virtually everything in the function will just convert to `any`.